src/styles/blocks/page.scss

```diff
@@ -0,0 +1,10 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+@import "../utils/vars";
```

Comment:
It would be better if all imports were in one place

Reply:
Thank you! All of your remarks make sense and I'll fix my issues.